internal/uvm/start.go

```diff
@@ -253,6 +253,13 @@ func (uvm *UtilityVM) Start(ctx context.Context) (err error) {
 		uvm.guestCaps = properties.GuestConnectionInfo.GuestDefinedCapabilities
 		uvm.protocol = properties.GuestConnectionInfo.ProtocolVersion
 	}
+
+	if uvm.cpuGroupID != "" {
```

Comment:
We should move this up to just after line 204. Otherwise the CPU group settings won't be applied during UVM boot. That could be an interesting toggle to support in the future, but for now I think it's better to just apply it as early as possible.